modules/minimum-required-permissions-ipi-gcp.adoc

```diff
@@ -189,6 +189,8 @@ If your organization’s security policies require a more restrictive set of per
 * `compute.firewalls.list`
 * `compute.forwardingRules.delete`
 * `compute.forwardingRules.list`
+* `compute.globalAddresses.list`
+* `compute.globalForwardingRules.list`
```

Comment:
I see you commented recently. If you are working off of master or the latest then the default install uses the cluster api. In this case yes it will be required. This particular card was mentioned when cluster api was not the default install path. Terraform was our resource creation, so we may need to adjust this linked bug to include cluster api.